src/languages/en.js

```diff
@@ -136,10 +136,10 @@ export default {
     hello: 'Hello',
     phoneCountryCode: '1',
     welcomeText: {
-        phrase1: 'Welcome to the New Expensify! Enter your phone number or email to continue.',
+        welcome: 'Welcome to the New Expensify! Enter your phone number or email to continue.',
         phrase2: 'Money talks. And now that chat and payments are in one place, it\'s also easy.',
         phrase3: 'Your payments get to you as fast as you can get your point across.',
-        phrase4: 'Welcome back to the New Expensify! Please enter your password.',
+        welcomeBack: 'Welcome back to the New Expensify! Please enter your password.',
```

Comment:
I'm not a fan of these names, especially when i read it in code like so,
```
`welcomeText.${showPasswordForm ? 'welcomeBack' : 'welcome'}`
```
Since its confusing to me why if showPasswordForm is true do we want welcomeBack versus welcome. Plus variables should be named after their intention of like what they store and not their parent/class names.

So in this case `welcome`, wants the phone or email so i would suggest - `enterPhoneOrEmail`. This becomes welcomeText.enterPhoneOrEmail which is a lot cleaner as welcomeText.welcome is redundant.
As for `welcomeBack`, it wants the password so i would suggest - `enterPassword`.

So now the logic becomes,
```
`welcomeText.${showPasswordForm ? 'enterPassword' : 'enterPhoneOrEmail'}`
```
which is a lot more verbose and cleaner imo.

Reply:
This is a NAB - not a blocker.

Reply:
That's a good point. In general, feels like the names for these translation keys is a bit all over the place. Wondering if we can maybe come up with some standard guidance for how to name them and then get everyone to apply it in a consistent way?

Reply:
> Wondering if we can maybe come up with some standard guidance for how to name them and then get everyone to apply it in a consistent way?

Yes, love it.